composer.json

```diff
@@ -21,23 +21,24 @@
   "require": {
     "php": "^7.4",
     "ext-json": "*",
-    "amphp/amp": "^2.1.2",
-    "amphp/http-client": "^4.0.0",
-    "amphp/byte-stream": "^1.6",
+    "amphp/amp": "^2.4.4",
+    "amphp/http-client": "^4.2.2",
+    "amphp/byte-stream": "^1.6.3",
     "amphp/file": "^1.0.0",
-    "amphp/socket": "^1.0.0",
-    "google/protobuf": "^3.9",
-    "prooph/event-store": "dev-master",
-    "psr/log": "^1.1.0"
+    "amphp/socket": "^1.1.1",
+    "google/protobuf": "^3.11.4",
+    "prooph/event-store": "dev-psalm",
```

Comment:
todo: set back to `dev-master` once https://github.com/prooph/event-store/pull/398 is merged.